src/Refitter.Core/RefitInterfaceImports.cs

```diff
@@ -4,41 +4,37 @@ namespace Refitter.Core;
 
 internal static class RefitInterfaceImports
 {
-    public static string[] GetImportedNamespaces(RefitGeneratorSettings settings) =>
-        settings.UseCancellationTokens
-            ? new[]
-            {
-                "Refit",
-                "System.Collections.Generic",
-                "System.Text.Json.Serialization",
-                "System.Threading",
-                "System.Threading.Tasks"
-            }
-            : new[]
-            {
-                "Refit",
-                "System.Collections.Generic",
-                "System.Text.Json.Serialization",
-                "System.Threading.Tasks"
-            };
+    private static string[] defaultNamespases = new[]
+    {
+        "Refit",
+        "System.Collections.Generic",
+        "System.Text.Json.Serialization",
+    };
+    public static string[] GetImportedNamespaces(RefitGeneratorSettings settings)
+    {
+        var namespaces = new List<string>(defaultNamespases);
+        if (settings.UseCancellationTokens)
+        {
+            namespaces.Add("System.Threading");
+        }
+
+        if (settings.ReturnIObservable)
+        {
+            namespaces.Add("System.Reactive");
```

Comment:
I'm not sure I fully understand why the generated code needs a dependency on `System.Reactive`. What do you need the `Unit` type for?

Reply:
Yes. System.Reactive is for Unit class. It is reactive "void" replacement.